scripts/provisioning/configcmd.py

```diff
@@ -143,7 +143,7 @@ def configure_openldap_replication(self):
 
         with open("hosts_list_file.txt", "w") as f:
           for node_machine_id in server_nodes_list:
-            hostname = self.get_confvalue(f'server_node>{node_machine_id}>hostname')
+            hostname = self.get_confvalue(f'server_node>{node_machine_id}>network>data>private_fqdn')
```

Comment:
change the variable name to node_private_fqdn

Reply:
Done